projects/fsastorefrontlib/src/cms-components/my-account/consent-management/consent-management.component.html

```diff
@@ -2,22 +2,6 @@
   <div class="container">
     <div class="row justify-content-center">
       <div class="col-md-6">
-        <div class="cx-consent-action-links text-right pb-4">
```

Comment:
Removed `accept all` and `remove all`, because it would add complexity to our implementation. Maybe we could make improvement task for this in the future.